arangod/Cluster/SynchronizeShard.cpp

```diff
@@ -913,10 +924,6 @@ bool SynchronizeShard::first() {
         return false;
       }
 
-      // This is necessary to accept replications from the leader which can
```

Comment:
It turns out that this can and must be set even later, namely, when we have got the exclusive lock (aka "hard read lock"), when we know the new `followingTermId`.